packages/application-shell/src/components/public-page-container/public-page-container.js

```diff
@@ -1,31 +1,96 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { css } from '@emotion/core';
 import { FormattedMessage } from 'react-intl';
-import { Spacings, Text } from '@commercetools-frontend/ui-kit';
+import {
+  Text,
+  Spacings,
+  customProperties,
+} from '@commercetools-frontend/ui-kit';
+import LogoBWWhiteSVG from '@commercetools-frontend/assets/images/ct_logo_bw_white.svg';
+import BackgroundImage from '@commercetools-frontend/assets/images/public-background.png';
 import PortalsContainer from '../portals-container';
-import styles from './public-page-container.mod.css';
 import messages from './messages';
 
 const year = new Date().getUTCFullYear();
 
-const PublicPageContainer = props => (
-  <React.Fragment>
-    <PortalsContainer />
-    <div className={styles.container}>
-      <Spacings.Stack scale="m">
-        {props.children}
-        <Spacings.Inline justifyContent="space-between">
-          <a href={`https://commercetools.com/privacy`} target="_blank">
-            <Text.Detail>
-              <FormattedMessage {...messages.privacyPolicy} />
-            </Text.Detail>
-          </a>
-          <Text.Detail>{`${year} © commercetools`}</Text.Detail>
-        </Spacings.Inline>
-      </Spacings.Stack>
-    </div>
-  </React.Fragment>
-);
+const PublicPageContainer = props => {
+  const privacyPolicyLink = (
+    <a href={`https://commercetools.com/privacy`} target="_blank">
+      <FormattedMessage {...messages.privacyPolicyLink} />
+    </a>
+  );
+
+  const termsOfServiceLink = (
+    <a href={`https://commercetools.com/terms`} target="_blank">
+      <FormattedMessage {...messages.termsOfServiceLink} />
+    </a>
+  );
+
+  return (
+    <React.Fragment>
+      <PortalsContainer />
+      <div
+        css={css`
+          height: 100vh;
```

Comment:
are we sure we want 100vh? won't the page scroll then? wouldn't it make more sense to subtract the height of the top bar?

Reply:
Hmm, but there is no top bar at the public page, or am I missing something?